packages/dashboard-frontend/src/services/helpers/types.ts

```diff
@@ -64,6 +64,10 @@ export enum DevWorkspaceStatus {
   STOPPING = 'Stopping',
 }
 
+export function isDevWorkspaceStatus(status: unknown): status is DevWorkspaceStatus {
+  return DevWorkspaceStatus[status as DevWorkspaceStatus] !== undefined;
```

Comment:
This implementation is not going to work as expected. 

I suppose to change this implementation to the next one:

```
export function isDevWorkspaceStatus(status: unknown): status is DevWorkspaceStatus {
  if (typeof status !== 'string') {
    return false;
  }

  return (
    Object.keys(DevWorkspaceStatus)
      .map(key => DevWorkspaceStatus[key])
      .indexOf(status) !== -1
  );
}
```

And we need tests
```
describe('DevWorkspaceStatus test', () => {
  test('DevWorkspaceStatus enum should include needed statuses', () => {
    const devWorkspaceStatusKeys = Object.keys(DevWorkspaceStatus).map(
      key => (DevWorkspaceStatus as { [key: string]: string })[key],
    );

    expect(devWorkspaceStatusKeys).toEqual([
      'Failed',
      'Failing',
      'Starting',
      'Terminating',
      'Running',
      'Stopped',
      'Stopping',
    ]);
  });

  test('isDevWorkspaceStatus should recognise all DevWorkspaceStatus keys', () => {
    expect(isDevWorkspaceStatus('Failed')).toBeTruthy();
    expect(isDevWorkspaceStatus('Failing')).toBeTruthy();
    expect(isDevWorkspaceStatus('Starting')).toBeTruthy();
    expect(isDevWorkspaceStatus('Terminating')).toBeTruthy();
    expect(isDevWorkspaceStatus('Running')).toBeTruthy();
    expect(isDevWorkspaceStatus('Stopped')).toBeTruthy();
    expect(isDevWorkspaceStatus('Stopping')).toBeTruthy();
  });

  test('isDevWorkspaceStatus should recognise wrong keys', () => {
    expect(isDevWorkspaceStatus('RUNNING')).toBeFalsy();
    expect(isDevWorkspaceStatus('STOPPING')).toBeFalsy();
    expect(isDevWorkspaceStatus('STOPPED')).toBeFalsy();
    expect(isDevWorkspaceStatus('STARTING')).toBeFalsy();
    expect(isDevWorkspaceStatus('ERROR')).toBeFalsy();
  });
});
```

WDYT?